components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -770,7 +770,10 @@ public enum ErrorMessages {
                         "application with name %s."),
         ERROR_CODE_ERROR_SHARING_APPLICATION_ROLE_CONFLICT("65149", "Organization %s has a non shared role with " +
                 "name %s.", "Server encountered an error while sharing application to organization %s " +
-                "due to a non shared role with name %s.");
+                "due to a non shared role with name %s."),
+        ERROR_CODE_ERROR_CHECKING_IS_ANCESTOR_ORGANIZATION("65150", "Unable to check if ancestor " +
```

Comment:
Let's fix the wording here

Reply:
fixed with 16963aa7d78d6ceee242379da54daa4ec6ec6ad1